consensus/src/quorum_store/proof_coordinator.rs

```diff
@@ -241,31 +275,25 @@ impl ProofCoordinator {
         Ok(None)
     }
 
-    fn update_counters_on_expire(
-        state: &IncrementalProofState,
-        validator_verifier: &ValidatorVerifier,
-    ) {
+    fn update_counters_on_expire(state: &IncrementalProofState) {
         // Count late votes separately
         if !state.completed && !state.self_voted {
             counters::BATCH_RECEIVED_LATE_REPLIES_COUNT.inc_by(state.voter_count());
             return;
         }
 
         counters::BATCH_RECEIVED_REPLIES_COUNT.observe(state.voter_count() as f64);
-        counters::BATCH_RECEIVED_REPLIES_VOTING_POWER
-            .observe(state.aggregate_voting_power(validator_verifier) as f64);
+        counters::BATCH_RECEIVED_REPLIES_VOTING_POWER.observe(state.aggregated_voting_power as f64);
         if !state.completed {
             counters::BATCH_SUCCESSFUL_CREATION.observe(0.0);
         }
     }
 
-    async fn expire(&mut self, validator_verifier: &ValidatorVerifier) {
+    async fn expire(&mut self) {
         let mut batch_ids = vec![];
         for signed_batch_info_info in self.timeouts.expire() {
             if let Some(state) = self.batch_info_to_proof.remove(&signed_batch_info_info) {
-                if !state.completed {
-                    batch_ids.push(signed_batch_info_info.batch_id());
-                }
+                Self::update_counters_on_expire(&state);
```

Comment:
This change does look ok, but let's carefully check the forge metrics. Previously when I screwed these metrics up, the forge tests would show a small number of batches that didn't get quorum, even though this wasn't true (because the batches were re-inited due to really late votes)